### PR TITLE
manual labour doesn't make you sleepy

### LINKED ISSUE #47391: Manual labor makes you sleepy
# Describe the bug
Certain manual labor tasks like mining with a pickaxe or chopping logs make the player sleepy.  This is most likely leftover behavior meant to simulate weariness from before actual weariness was implemented.

# Steps To Reproduce
1. Debug in a chainsaw
2. Spend half a day chopping up a bunch of logs
3. Observe that you're really sleepy even though it's only noon.

# Expected behavior
Manual labor makes you extremely weary but isn't so boring that it makes you want to fall asleep

# Versions and configuration
- OS: Linux
    - OS Version: Distributor ID: Ubuntu; Description: Ubuntu 20.04.2 LTS; Release: 20.04; Codename: focal; 
- Game Version: 0.E-9545-g034ac1589c [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Blaze Industries [blazeindustries],
    Magiclysm [magiclysm],
    Stats Through Skills [StatsThroughSkills]
]

# Additional context
The relevant code is in `activity_handlers.cpp`.  Numerous activities call `p->mod_fatigue()` which increases sleepiness rather than weariness.  Gaining weariness is handled automatically by activity levels so most of these calls can probably just be removed entirely.  Calls to `mod_thirst()` and `mod_stored_nutr()` should similarly be audited as well.

### PR DESCRIPTION
**Pull request by s7eamy**
#### Summary 
Bugfixes "removed mod_fatigue(), mod_stored_nutr(), mod_thirst() calls from various activities"

#### Purpose of change
Fixes #47391

#### Describe the solution
by removing the functions calls, manual labour doesn't make you sleepy anymore

### CODE REVIEW

File: src/activity_handlers.cpp [hunk 1]
```diff
@@ -1821,23 +1821,11 @@ void activity_handlers::pickaxe_finish( player_activity *act, player *p )
...
-            p->mod_thirst( 15 - ( helpersize * 3 ) );
-            if( p->has_trait( trait_STOCKY_TROGLO ) ) {
-                // Yep, dwarves can dig longer before tiring
-                p->mod_fatigue( 20 - ( helpersize  * 3 ) );
-            } else {
-                p->mod_fatigue( 30 - ( helpersize  * 3 ) );
-            }
+            // Tunneling through solid rock is sweaty, backbreaking work
+            // Betcha wish you'd opted for the J-Hammer
             p->mod_pain( std::max( 0, ( 2 * static_cast<int>( rng( 1, 3 ) ) ) - helpersize ) );
```
Comment (actual-nh) [inline src/activity_handlers.cpp:1826]:
```suggestion
            if( p->has_trait( trait_STOCKY_TROGLO ) ) {
                p->mod_pain( std::max( 0, ( 1 * static_cast<int>( rng( 0, 3 ) ) ) - helpersize ) );
            } else {
                p->mod_pain( std::max( 0, ( 2 * static_cast<int>( rng( 1, 3 ) ) ) - helpersize ) );
            }
```
I am suggesting this as a matter of game balance - there is little enough already making the Troglodyte mutation path attractive, and they are supposed to function better underground.

File: src/activity_handlers.cpp [hunk 2]
```diff
@@ -1821,23 +1821,11 @@ void activity_handlers::pickaxe_finish( player_activity *act, player *p )
...
         } else if( here.move_cost( pos ) == 2 && here.get_abs_sub().z == 0 &&
                    here.ter( pos ) != t_dirt && here.ter( pos ) != t_grass ) {
```
Comment (actual-nh) [inline src/activity_handlers.cpp:1828]:
```suggestion

```
This should fix the LGTM error, as you probably have already figured out...

File: src/activity_handlers.cpp (hunk 1 above)
Comment (actual-nh) [reply src/activity_handlers.cpp:1826]: BTW, ping: @anothersimulacrum - your thoughts?

Comment (anothersimulacrum) [reply src/activity_handlers.cpp:1826]: I don't have any opinion on this.

Comment (s7eamy) [reply src/activity_handlers.cpp:1826]: Should I commit this suggestion, then? It doesn't really belong in the "Bugfixes" category of this PR, more like "Balance" (if that matters haha)

Comment (actual-nh) [reply src/activity_handlers.cpp:1826]: When I get a chance, or if you want to do it, I'll check the git "blame" for who originally put in the conditional, and see about "pinging" them for their opinion. In the meantime, you might put "[CR]" (Comments Requested) at the start of the PR's title (you can modify it at the top of the PR, with the edit button beside the title).

Comment (actual-nh) [reply src/activity_handlers.cpp:1826]: Ping: @John-Candlebury, @I-am-Erk, @KorGgenT? (Going off of #38505 - original account deleted, unfortunately, but these were those that were asked to review.)

**Comment by s7eamy** [comment]
what should I do about this test failing?
```
../tests/field_test.cpp:357: FAILED:
(~[slow] ~[.])=>	  CHECK( fields_test_duration() < time_limit )
(~[slow] ~[.])=>	with expansion:
Error: (~[slow] ~[.])=>	  {?} < {?}
(~[slow] ~[.])=>	with message:
(~[slow] ~[.])=>	  Terrain should be irradiated under 5 minutes
```

**Comment by actual-nh** [comment]
> what should I do about this test failing?

Sigh... it isn't your doing; some of the fields tests have some randomness involved (whether something spreads to another tile, in this case), so they fail (probably more often than they should, but that's a matter for another PR).

File: src/activity_handlers.cpp (hunk 1 above)
Comment (actual-nh) [reply src/activity_handlers.cpp:1826]: I just checked the history on STOCKY_TROGLO as a mutation. Ping @KA101, who created it; the tagline for [the commit](https://github.com/CleverRaven/Cataclysm-DDA/commit/05d92e3bc412826d2c9a94a97b29447bf9254137) adding STOCKY_TROGLO is "Get digging!".

Comment (anothersimulacrum) [reply src/activity_handlers.cpp:1826]: > Should I commit this suggestion, then? It doesn't really belong in the "Bugfixes" category of this PR, more like "Balance" (if that matters haha)

You're preserving some behaviour of STOCK_TROGLO being better at digging, I think it's fine to do here.

**Comment by kevingranade** [comment]
Looks good to me with or without the trog adjustment, just need a quick test to verify that these activities are still suitably exhausting after the adjustment.

**Comment by actual-nh** [comment]
Looks like "CR" can be removed. @kevingranade: What is your definition of "adequately fatiguing"?

**Comment by s7eamy** [comment]
hey, this actually merged! thanks a lot for your help @actual-nh, i appreciate it a lot!!

**Pull request merged**